dex/testing/btc/base-harness.sh

```diff
@@ -73,7 +73,7 @@ EOF
 ################################################################################
 
 tmux rename-window -t $SESSION:0 'alpha'
-tmux send-keys -t $SESSION:0 "set +o history" C-m
+tmux send-keys -t $SESSION:0 "set +o nolog" C-m
```

Comment:
Two things:
1. `nolog` has the opposite semantics from `history`, so to stop recording history it would use a `-o nolog` instead of `+o`.
2. `nolog` is apparently ignored by bash, so I don't understand why `history` isn't working for you.  Can you double check this and the bash version you have?

Snip from my bash man page:

```
                      noglob  Same as -f.
                      nolog   Currently ignored.
                      notify  Same as -b.
                      nounset Same as -u.
```

And from https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html:

```
noglob

    Same as -f.
nolog

    Currently ignored.
notify

    Same as -b.
```